Issue after upgrading to 1.0.4

Failed to compile.

```
multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/src/runtime/ReactRefreshEntry.js ./node_modules/react-dev-utils/webpackHotDevClient.js .
Module not found: Can't resolve '/own_projects/project-name' in '/own_projects/project-name'
```

I also tried removing `node_modules` and `package-lock.json`, and the issue are still there.

@dariocravero I am not able to install dependencies in the example project you provided. Looks like there's something wrong with the `package-lock.json`. Can you check it out?

```
~/work/cra-react-refresh-reproduce master
❯ npm i

> fsevents@1.2.13 install /Users/ethan/work/cra-react-refresh-reproduce/node_modules/jest-haste-map/node_modules/fsevents
> node install.js

internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module 'nan'
Require stack:
- /Users/ethan/work/cra-react-refresh-reproduce/node_modules/jest-haste-map/node_modules/fsevents/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:120:20)
    at Object.runInThisContext (vm.js:311:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at evalScript (internal/process/execution.js:94:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/ethan/work/cra-react-refresh-reproduce/node_modules/jest-haste-map/node_modules/fsevents/[eval]'
  ]
}
gyp: Call to 'node -e "require('nan')"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/ethan/.nvm/versions/node/v12.17.0/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Darwin 19.5.0
gyp ERR! command "/Users/ethan/.nvm/versions/node/v12.17.0/bin/node" "/Users/ethan/.nvm/versions/node/v12.17.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/ethan/work/cra-react-refresh-reproduce/node_modules/jest-haste-map/node_modules/fsevents
gyp ERR! node -v v12.17.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok

> fsevents@1.2.13 install /Users/ethan/work/cra-react-refresh-reproduce/node_modules/watchpack-chokidar2/node_modules/fsevents
> node install.js

internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module 'nan'
Require stack:
- /Users/ethan/work/cra-react-refresh-reproduce/node_modules/watchpack-chokidar2/node_modules/fsevents/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:120:20)
    at Object.runInThisContext (vm.js:311:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at evalScript (internal/process/execution.js:94:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/ethan/work/cra-react-refresh-reproduce/node_modules/watchpack-chokidar2/node_modules/fsevents/[eval]'
  ]
}
gyp: Call to 'node -e "require('nan')"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/ethan/.nvm/versions/node/v12.17.0/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Darwin 19.5.0
gyp ERR! command "/Users/ethan/.nvm/versions/node/v12.17.0/bin/node" "/Users/ethan/.nvm/versions/node/v12.17.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/ethan/work/cra-react-refresh-reproduce/node_modules/watchpack-chokidar2/node_modules/fsevents
gyp ERR! node -v v12.17.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok

> fsevents@1.2.13 install /Users/ethan/work/cra-react-refresh-reproduce/node_modules/webpack-dev-server/node_modules/fsevents
> node install.js

internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module 'nan'
Require stack:
- /Users/ethan/work/cra-react-refresh-reproduce/node_modules/webpack-dev-server/node_modules/fsevents/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:120:20)
    at Object.runInThisContext (vm.js:311:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at evalScript (internal/process/execution.js:94:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/ethan/work/cra-react-refresh-reproduce/node_modules/webpack-dev-server/node_modules/fsevents/[eval]'
  ]
}
gyp: Call to 'node -e "require('nan')"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/ethan/.nvm/versions/node/v12.17.0/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Darwin 19.5.0
gyp ERR! command "/Users/ethan/.nvm/versions/node/v12.17.0/bin/node" "/Users/ethan/.nvm/versions/node/v12.17.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/ethan/work/cra-react-refresh-reproduce/node_modules/webpack-dev-server/node_modules/fsevents
gyp ERR! node -v v12.17.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok

> core-js@2.6.11 postinstall /Users/ethan/work/cra-react-refresh-reproduce/node_modules/babel-runtime/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js@3.6.5 postinstall /Users/ethan/work/cra-react-refresh-reproduce/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js-pure@3.6.5 postinstall /Users/ethan/work/cra-react-refresh-reproduce/node_modules/core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"

added 1678 packages from 816 contributors and audited 1680 packages in 15.513s

59 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
```

--- a/index.js
+++ b/index.js
@@ -6,7 +6,7 @@ const webpackDevClientEntry = require.resolve(
 const paths = require("react-scripts/config/paths");
 
 const overrideEntry = () => (config) => {
-  config.entry = paths.appPath;
+  config.entry = paths.appIndexJs;
   return config;
 };
 

